audit/projects/kubernetes-public/services/bigquery/bigquery.datasets.k8s_prod_readiness.access.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "role": "WRITER",
+    "specialGroup": "projectWriters"
+  },
+  {
+    "role": "OWNER",
+    "specialGroup": "projectOwners"
+  },
+  {
+    "role": "OWNER",
+    "userByEmail": "spiffxp@google.com"
+  },
+  {
+    "role": "READER",
+    "specialGroup": "projectReaders"
+  }
+]
```

Comment:
This is incorrect because https://github.com/kubernetes/k8s.io/pull/2717 hasn't been fully deployed